modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -784,20 +785,26 @@ public void remove(@NotNull Throwable e) {
                     }
             );
         }).exceptionally(t -> {
-            Throwable ex = t.getCause();
+            Throwable cause = t.getCause();
```

Comment:
What a mess, I think this code deserves some comments as well

Reply:
I wrote a little more code, I hope it became clearer.